src/plugins/ui_feature/renderer.ts

```diff
@@ -341,7 +341,7 @@ export class RendererPlugin extends UIPlugin {
     const y = box.y + 1;
     const textHeight = computeTextLinesHeight(textLineHeight, numberOfLines);
     const hasEnoughSpaces = box.height > textHeight + MIN_CELL_TEXT_MARGIN * 2;
-    const verticalAlign = box.verticalAlign || "middle";
```

Comment:
You should also change the default in the `o-tool` vertical align of the topbar. And we should have a constant for the default align, but I think @laa-odoo is doing it in topbar refactoring task, right ?

Reply:
There is no indication in the dropdown, but the tool change based on the alignment of the selected cell
![image](https://user-images.githubusercontent.com/31045223/228271209-4d020ef5-64fb-429e-b82a-e374facee680.png)
![image](https://user-images.githubusercontent.com/31045223/228271238-30fc2b29-e341-4afd-8d98-3635d5a030c1.png)